src/shared/constants.ts

```diff
@@ -24,42 +25,53 @@ export const MICROBLOCKS_ENABLED = !IS_TEST_ENV && true;
 export const GITHUB_ORG = 'hirosystems';
 export const GITHUB_REPO = 'stacks-wallet-web';
 
-const DEFAULT_MAINNET_SERVER = 'https://stacks-node-api.stacks.co';
-export const DEFAULT_TESTNET_SERVER = 'https://stacks-node-api.testnet.stacks.co';
-
-export enum DefaultNetworkIds {
+export enum DefaultNetworkConfigurationIdsAvailableInWallet {
   mainnet = 'mainnet',
   testnet = 'testnet',
   devnet = 'devnet',
 }
 
-export type DefaultNetworkNames = keyof typeof DefaultNetworkIds;
+export type DefaultNetworkConfigurationsAvailableInWallet =
```

Comment:
Name suggestion was just to explain the point, reckon we go with `DefaultNetworkConfigurations`, think it's clear with separation between network{mode,config}

Reply:
I know you love long names! 😆